common/src/test/kotlin/com/hartwig/actin/molecular/evidence/known/KnownEventResolverFactoryTest.kt

```diff
@@ -14,18 +15,18 @@ class KnownEventResolverFactoryTest {
 
     @Test
     fun `Should filter known events`() {
-        val hotspot1 = TestServeKnownFactory.hotspotBuilder().addSources(firstAllowedSource(), Knowledgebase.UNKNOWN).build()
-        val hotspot2 = TestServeKnownFactory.hotspotBuilder().addSources(firstAllowedSource()).build()
+        val hotspot1 = TestServeKnownFactory.hotspotBuilder().addSources(PRIMARY_KNOWN_EVENT_SOURCE, Knowledgebase.UNKNOWN).build()
+        val hotspot2 = TestServeKnownFactory.hotspotBuilder().addSources(PRIMARY_KNOWN_EVENT_SOURCE).build()
         val hotspot3 = TestServeKnownFactory.hotspotBuilder().addSources(Knowledgebase.UNKNOWN).build()
         val knownEvents = ImmutableKnownEvents.builder().addHotspots(hotspot1, hotspot2, hotspot3).build()
 
-        val filtered = KnownEventResolverFactory.filterKnownEvents(knownEvents)
-        assertThat(filtered.hotspots().size.toLong()).isEqualTo(2)
-        assertThat(filtered.hotspots().contains(hotspot1)).isTrue()
-        assertThat(filtered.hotspots().contains(hotspot2)).isTrue()
-    }
+        val primaryKnownEvents = KnownEventResolverFactory.filterKnownEvents(knownEvents, true)
+        assertThat(primaryKnownEvents.hotspots().size.toLong()).isEqualTo(2)
```

Comment:
You could replace this with ```assertThat(primaryKnownEvents.hotspots()).hasSize(2)```

Or even better, you can replace the 3 lines with 
```
assertThat(primaryKnownEvents.hotspots()).containsExactlyInAnyOrder(hotspot1, hotspot2)
```
since this also asserts the size implicitly